### ISSUE TITLE
confused with loss fuction

### ISSUE THREAD

**Issue by yl-jiang** [uvipen/Yolo-v2-pytorch#15]
thanks to your great work, I learn a lot from your code, but when I use your code to train VOC2007 from scratch the confidence loss very large, so the total loss mostly control by confidence loss,I don't know how to fix it, can you give me some suggestions?

**Comment by ArthurMoreau** [comment]
I have exactly the same issue.
Did you find a way to handle it ?

**Comment by AlbertMa123** [comment]
i got the same problem. when i train the model for 250 epoches, the loss is Loss:0.32 (Coord:0.00 Conf:0.31 Cls:0.01). The conf is the largest loss.
